src/App.tsx

```diff
@@ -28,129 +53,18 @@ export const App: React.FC = () => {
           </form>
         </header>
 
```

Comment:
header part can be separated from app, for example you can create NewTodo component

Reply:
Already did it during the next task, so yes, I can just fix it in here also